src/js/components/help/downloads.js

```diff
@@ -0,0 +1,323 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { connect } from 'react-redux';
+import copy from 'copy-to-clipboard';
+import Cookies from 'universal-cookie';
+
+import { Accordion, AccordionDetails, AccordionSummary, Chip, Menu, MenuItem, Typography } from '@mui/material';
+import { ArrowDropDown, ExpandMore, FileDownloadOutlined as FileDownloadIcon, Launch } from '@mui/icons-material';
+
+import { setSnackbar } from '../../actions/appActions';
+import { getCurrentUser, getDocsVersion, getIsEnterprise, getTenantCapabilities } from '../../selectors';
+import Time from '../common/time';
+import { getToken } from '../../auth';
+import { detectOsIdentifier } from '../../helpers';
+import Tracking from '../../tracking';
+
+// wget --auth-no-challenge --user "$HOSTED_MENDER_EMAIL" --password "$HOSTED_MENDER_PASSWORD" https://downloads.customer.mender.io/content/hosted/mender-monitor/debian/1.2.0/mender-monitor_1.2.0-1%2Bdebian%2Bbuster_all.deb
```

Comment:
Leftover comment?

Reply:
leftover reminder to add `wget` in the download options... - fixed now 👍